styles/main.css

```diff
@@ -48,39 +48,49 @@ header nav {
   margin: 0em 2em;
 }
 
-ul li {
+header nav ul li {
   display: inline;
 }
```

Comment:
Make sure your selectors are specific enough.  As you had it, ALL unordered list items on your website would be displayed as inline elements